### PR TITLE
First draft

### PR DESCRIPTION
**Pull request by Ross-Alexandra**
Merge the first draft into staging for final testing before a prod deploy.

### CODE REVIEW

File: theserverproject/src/pages/Home/components/Footer/Footer.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
+
+/* Footer IDs  */
+#footer {
+    width: 100vw;
+    height: var(--footer-height);
+    background-color: var(--text-bg-color);
+    display: flex;
+    max-width: 100vw;
```
Comment (Ross-Alexandra-Review) [inline theserverproject/src/pages/Home/components/Footer/Footer.css]: Is this needed?

File: theserverproject/src/minorComponents/dropdown/Dropdown.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    overflow: visible;
+    align-self: flex-start;
+}
+
+/* Content div */
+.dropdown-content {
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    /* height of the navbar - 5 margin - 8 padding. */
```
[the remark this answers is not in the document]
Comment (Ross-Alexandra) [reply theserverproject/src/minorComponents/dropdown/Dropdown.css]: Addressed with 307f489

File: theserverproject/src/minorComponents/dropdown/Dropdown.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    border-style: solid;
+    border-top-style: none;
+    border-color: var(--text-fg-color);
+
+    /* Default item css */
+    text-align: center;
+}
+
+/* Content items */
+.dropdown-item {
```
[the remark this answers is not in the document]
Comment (Ross-Alexandra) [reply theserverproject/src/minorComponents/dropdown/Dropdown.css]: Addressed with 307f489

File: theserverproject/src/minorComponents/dropdown/Dropdown.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        );
+
+        let dropdownButtonClass = (this.props.dropdownTitleClassOverride ?
+                this.props.dropdownTitleClassOverride + ' astext ' + this.state.defaultDropdownTitleClass
+            :
+                'astext ' + this.state.defaultDropdownTitleClass
+        );
+
+        let dropdownContentClass = this.props.dropdownContentClassOverride ? this.props.dropdownContentClassOverride + ' dropdown-content' : 'dropdown-content';
+  //      let dropdownContentClass = 'dropdown-content';
```
[the remark this answers is not in the document]
Comment (Ross-Alexandra) [reply theserverproject/src/minorComponents/dropdown/Dropdown.js]: Addressed with 307f489

File: theserverproject/src/minorComponents/image/Image.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    // Fallback text classes
+    let fallbackText = props.defaultText ? props.defaultText : "No Image Found";
+    let fallbackTextClass = props.textClassOverride ? props.textClassOverride + ' text-fallback' : 'text-fallback';
+
+    // Image classes
+    let imageAltText = props.altText ? props.altText : "";
+    let imageClass = props.imageClassOverride ? props.imageClassOverride + ' image-image' : 'image-image';
+
+    if (!props.image) {
+        return <p className={ fallbackTextClass}>{fallbackText}</p>;
```
[the remark this answers is not in the document]
Comment (Ross-Alexandra) [reply theserverproject/src/minorComponents/image/Image.js]: Addressed with 307f489

File: theserverproject/src/pages/Home/components/Footer/Footer.css (hunk 1 above)
Comment (Ross-Alexandra) [reply theserverproject/src/pages/Home/components/Footer/Footer.css]: Addressed with 307f489

File: theserverproject/src/minorComponents/dropdown/Dropdown.js [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            dropdownVisible: false,
+            defaultDropdownTitleClass: 'dropdown-title',
+        };
+
+        this.titleRef = React.createRef();
```
[the remark this answers is not in the document]
Comment (Ross-Alexandra) [reply theserverproject/src/minorComponents/dropdown/Dropdown.js]: Addressed with 307f489

**Pull request merged**